docsource/modules130-140.rst

```diff
@@ -104,7 +104,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |contacts                                    | Nothing to do                                   |
 +--------------------------------------------+-------------------------------------------------+
-| |new| coupon                               |                                                 |
+| |new| coupon                               | Split from sale_coupon                          |
```

Comment:
We should prefix it with `Done` or `Nothing to do` to be handled by CI.

Reply:
But that's when the full script will be added. Not now.

Reply:
Well, I removed that annotation to avoid confusions.